mappings/net/minecraft/world/gen/surfacebuilder/SurfaceRules.mapping

```diff
@@ -1,8 +1,15 @@
 CLASS net/minecraft/unmapped/C_flergppe net/minecraft/world/gen/surfacebuilder/SurfaceRules
 	FIELD f_buxwpbgc UNDER_FLOOR Lnet/minecraft/unmapped/C_flergppe$C_mecsxqth;
 	FIELD f_oemhkmjb UNDER_CEILING Lnet/minecraft/unmapped/C_flergppe$C_mecsxqth;
+	FIELD f_rgnrkvqz DEEP_UNDER_FLOOR Lnet/minecraft/unmapped/C_flergppe$C_mecsxqth;
 	FIELD f_rwakgupc ON_CEILING Lnet/minecraft/unmapped/C_flergppe$C_mecsxqth;
 	FIELD f_uofupvny ON_FLOOR Lnet/minecraft/unmapped/C_flergppe$C_mecsxqth;
+	FIELD f_usgtpwrh VERY_DEEP_UNDER_FLOOR Lnet/minecraft/unmapped/C_flergppe$C_mecsxqth;
```

Comment:
This is super funny but super vague, what does it mean that it is VERY_DEEP_UNDER_FLOOR like what does that imply. We already have DEEP_UNDER_FLOOR so this name would be very confusing.

Reply:
Any suggestions then?

Reply:
I suggest probably DEEPEST_LEVEL_UNDER_FLOOR or DEEPEST_LEVEL maybe considering its the last one